actions/notification_actions.jsx

```diff
@@ -168,6 +198,7 @@ const notifyMe = (title, body, channel, teamId, silent, soundName) => (dispatch,
 
         if (isDesktopApp() && window.desktop && semver.gte(window.desktop.version, '4.6.0')) {
             msg.data = {soundName};
+            msg.url = url;
```

Comment:
4.6 doesn't understand this parameter, it should at least be version '4.7.2' if we backport the change.